content/blog/2018/egress-monitoring-access-control.md

```diff
@@ -0,0 +1,546 @@
+
+To achieve that goal, the organization's operations people will monitor the access to the external services and will analyze the Istio logs to verify that no unauthorized request was sent to [edition.cnn.com/politics](https://edition.cnn.com/politics). They will also configure Istio to prevent access to [edition.cnn.com/politics](https://edition.cnn.com/politics) automatically.
+
+The organization is resolved to prevent any tampering with the new policy. It decided to put mechanisms in place that will prevent any possibility for a malicious application to access the forbidden topic.
+
+## Related tasks
+
+The [Control Egress Traffic](/docs/tasks/traffic-management/egress/) task demonstrates how external (outside the Kubernetes cluster) HTTP and HTTPS services can be accessed from applications inside the mesh. The [TLS Origination for Egress Traffic](/docs/tasks/traffic-management/egress-tls-origination/) task demonstrates how to allow the applications to send HTTP requests to external servers that require HTTPS. The [Configure an Egress Gateway](/docs/tasks/traffic-management/egress-gateway/) task describes how to configure Istio to direct egress traffic through a dedicated gateway service called _egress gateway_.
+
+The [Collecting Metrics and Logs](/docs/tasks/telemetry/metrics-logs/) task describes how to configure metrics and logs for services in a mesh. The [Visualizing Metrics with Grafana](/docs/tasks/telemetry/using-istio-dashboard/) describes the Istio Dashboard to monitor mesh traffic. The [Basic Access Control](/docs/tasks/security/basic-access-control/) task shows how to control access to in-mesh services. The [Secure Access Control](http://localhost:1313/docs/tasks/security/secure-access-control/) task shows how to configure access policies using black or white list checkers. As opposed to the telemetry and security tasks above, this blog post describes Istio's monitoring and access policies applied exclusively to the egress traffic.
```

Comment:
Maybe this and the previous paragraph would work better in a bullet list?

Reply:
@geeknoid Please see my comment https://github.com/istio/istio.github.io/pull/1570#discussion_r197482369. Your call.